src/sentry/search/eap/uptime_check_columns.py

```diff
@@ -0,0 +1,93 @@
+from sentry_protos.snuba.v1.request_common_pb2 import TraceItemType
+
+from sentry.search.eap import constants
+from sentry.search.eap.columns import (
+    ColumnDefinitions,
+    ResolvedColumn,
+    VirtualColumnDefinition,
+    datetime_processor,
+    project_context_constructor,
+    project_term_resolver,
+)
+from sentry.search.eap.common_columns import COMMON_COLUMNS
+
+UPTIME_CHECK_ATTRIBUTE_DEFINITIONS = {
+    column.public_alias: column
+    for column in COMMON_COLUMNS
+    + [
+        ResolvedColumn(
+            public_alias="environment",
+            internal_name="sentry.environment",
+            search_type="string",
+        ),
```

Comment:
We have some shortcut functions for this kind of thing if you'd like to use it/add to it

https://github.com/getsentry/sentry/blob/8279a70bfe9666ea51f8685032efc2f17ef22969/src/sentry/search/eap/columns.py#L150-L153

Reply:
thanks for calling out. see https://github.com/getsentry/sentry/pull/85124. in uptime's schema, we do not use the `sentry.` prefixes